# avoid duplicate definition of enum plist_format_t introduced in libplist 2.3.0
# adapted from patch for libimobiledvice at: https://github.com/archlinux/svntogit-packages/blob/packages/libimobiledevice/trunk/libplist-2.3.0.diff
# 
--- src/utils.c.orig	2020-03-19 18:59:46.000000000 +0000
+++ src/utils.c	2023-05-21 10:11:13.413337184 +0000
@@ -302,7 +302,7 @@
 	return 1;
 }
 
-int plist_write_to_filename(plist_t plist, const char *filename, enum plist_format_t format)
+int plist_write_to_filename(plist_t plist, const char *filename, plist_format_t format)
 {
 	char *buffer = NULL;
 	uint32_t length;
--- src/utils.h.orig	2020-03-19 18:59:46.000000000 +0000
+++ src/utils.h	2023-05-21 10:10:59.970236793 +0000
@@ -78,13 +78,8 @@
 int buffer_read_from_filename(const char *filename, char **buffer, uint64_t *length);
 int buffer_write_to_filename(const char *filename, const char *buffer, uint64_t length);
 
-enum plist_format_t {
-	PLIST_FORMAT_XML,
-	PLIST_FORMAT_BINARY
-};
-
 int plist_read_from_filename(plist_t *plist, const char *filename);
-int plist_write_to_filename(plist_t plist, const char *filename, enum plist_format_t format);
+int plist_write_to_filename(plist_t plist, const char *filename, plist_format_t format);
 
 uint64_t mstime64(void);
 void get_tick_count(struct timeval * tv);
